package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "pulsar-client",
-  "version": "1.8.1-rc.0",
+  "version": "1.8.0-rc.0",
```

Comment:
Can you rebase master? In the master, the version was originally 1.8.0-rc.0

https://github.com/apache/pulsar-client-node/blob/5173415d03e774afceb3d89bd90c2f20bc5a0f0e/package-lock.json#L2-L3